.env.development

```diff
@@ -13,4 +13,3 @@ INNLOGGINGSSTATUS_URL=http://localhost:8095/innloggingsstatus/auth
 REVALIDATOR_PROXY_ORIGIN=http://localhost:3002
 XP_ORIGIN=http://localhost:8080
 IS_FAILOVER_INSTANCE='false'
-BUILD_ID=development
```

Comment:
Tror denne var nødvendig for å få korrekte cache-url'er for oversiktssider i dev-modus

Reply:
Skjønner ikke helt hvorfor jeg har rørt den. Sorry